xtask/src/docgen.rs

```diff
@@ -48,6 +54,67 @@ pub fn typable_commands() -> Result<String, DynError> {
     Ok(md)
 }
 
+pub fn static_commands() -> Result<String, DynError> {
+    let mut md = String::new();
+    let keymaps = [
+        EditorView::default().keymaps.map()[&Mode::Normal].reverse_map(),
+        EditorView::default().keymaps.map()[&Mode::Select].reverse_map(),
+        EditorView::default().keymaps.map()[&Mode::Insert].reverse_map(),
+    ];
```

Comment:
For this slice we can use a tuple for each element so that we have the mode name associated to its reverse map:

```rust
let keymap = EditorView::default().keymaps.map();
let keymaps = [
    ("normal", keymap[&Mode::Normal].reverse_map()),
    // same for select & insert
];
```

That way we don't need to lookup the position of the keymap to get the mode below.